css/templates/lune/components/_header.css

```diff
@@ -0,0 +1,51 @@
+@import "../page-vars";
+
+.lune .ampstart-headerbar {
+  display: flex;
+  align-items: center;
+
+  height: var(--layout-header-height-m);
+
+  box-shadow: none;
+
+  will-change: transform;
+}
+
+.lune .ampstart-headerbar-title {
+  position: absolute;
+  top: 50%;
+  left: 50%;
+
+  font-size: 1.3125rem; /* 21px */
+  font-weight: var(--font-weight-black);
+
+  letter-spacing: .52em;
```

Comment:
We're using ems here so the letter spacing so it will scale proportionally to the font size. It'll save us from adding additional styles if the font size increases.